packages/ramldoc-generator/src/main.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const path = require('path');
+const shell = require('shelljs');
+const chalk = require('chalk');
+
+function generateRaml(options) {
+  const resolvedSourcePath = path.resolve(options.apiSpecSourcePath);
```

Comment:
Can you explain the reason for using `path.resolve`?

Reply:
I prefer to work with absolute path, in case a user passes a relative path.

Reply:
Ok thanks